source/content/guides/secrets/06-troubleshooting.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Troubleshooting
+description: Securely store secrets in the Pantheon Platform.
+terminuspage: true
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/troubleshooting
+reviewed: "2024-05-01"
+---
+## Integrated Composer Build fails on private packages
+
+This is the most common error when sites are using secrets and Integrated Composer. This may manifest in different ways and may be caused by different problems. This playbook tries to cover them.
+
+**Error getting a private package during the IC build**
+
+This is the most common error; an example message associated to this error is:
+
+- Failed to download vendor/package from dist (where vendor/package may be any private package)
+
+Some possible causes for this error:
+
+<dl>
+<dt>
+
+**Problem:** Secrets are not correctly set for the site. Secrets for IC should be of scope ic and type composer. See instructions in the plugin README: [https://github.com/pantheon-systems/terminus-secrets-manager-plugin#mechanism-1-oauth-composer-authentication](https://github.com/pantheon-systems/terminus-secrets-manager-plugin#mechanism-1-oauth-composer-authentication)
```

Comment:
This could reference the docs page rather than the plugin README now.

Reply:
> **Problem:** Secrets are not correctly set for the site. Secrets for Integrated Composer to use need to be type `composer` and have scope `ic`. Secret types and scopes are covered in the [Basic Concepts](/guides/secrets/02-basic-concepts) documentation.